docs-raw/site/docs/fabric-gateway-features.md

```diff
@@ -75,8 +76,18 @@ spec:
     /resources:
       get: {}
 ```
+
 In the above example, we are not defining any services (and it is illegal to define services via the `x-fabric-service` key if you have the `x-external-service-provider` set) in the Gateway resource. Instead, we are saying that these services will be provided by a [StackSet](https://cloud.docs.zalando.net/tutorials/migrating-from-stups/#recommended-stacksets) operator. The created gateway will not generate any ingress resources until the named StackSet is present and has service details available.
 
+#### Versioned Hosts
+
+When using the stackset integration, the gateway will also configure a host for accessing each stack individually, ignoring traffic-switching. For example, if you have stacks `my-stack-v1` and `my-stack-v2` and the gateway operator has set `VERSIONED_HOSTS_BASE_DOMAIN` to `abc.com`, you will be able to access the stacks at `https://my-stack-v1.abc.com` and `https://my-stack-v2.abc.com` respectively.
```

Comment:
`VERSIONED_HOSTS_BASE_DOMAIN` isn't user configurable, no? It might be confusing to end users reading about it. Maybe have that in the install docs as opposed to the user docs

Reply:
That's why I mentioned the gateway operator though, there's no way for me to describe the urls without referring to it so I think it has to stay here. The base domain env was described at `fabric-gateway-features.md` but I can add it to the install docs as well if you want

Reply:
In my head features docs should relate to stuff that resource creators have control over. If they aren't able to change this value, I don't think they should care about the env var name. We'll be specifically setting this to their cluster domain for them.

But if ye both think it's fine, work away...
I'm only trying to predict potential confusion atm... 🤷‍♂

Reply:
Meant to say it was described at https://github.com/zalando-incubator/fabric-gateway#feature-flags

> In my head features docs should relate to stuff that resource creators have control over

Hmm in that case I'd just remove the bit saying what the url will be. I think that would be more confusing possibly. I'd hope the base url would be obvious to anyone who is using the gateway